pkg/test/storage.go

```diff
@@ -92,10 +95,13 @@ func StorageClass(overrides ...StorageClassOptions) *storagev1.StorageClass {
 	if options.Zones != nil {
 		allowedTopologies = []v1.TopologySelectorTerm{{MatchLabelExpressions: []v1.TopologySelectorLabelRequirement{{Key: v1.LabelTopologyZone, Values: options.Zones}}}}
 	}
+	if options.Provisioner == nil {
```

Comment:
Consider just checking "" and not making options.Provisioner a pointer to avoid type juggling.